lib/vsc/install/shared_setup.py

```diff
@@ -1357,6 +1357,7 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
             'setuptools'
         ],
         'excluded_pkgs_rpm': [], # vsc-install ships vsc package (the vsc package is removed by default)
+        'namespace_packages': ['vsc'],
```

Comment:
as said before, for `vsc-install`, `vsc` is not a pure namespace; but it should be set as default for all other vsc packages